cstar/tests/config.py

```diff
@@ -0,0 +1,70 @@
+"""
+Configuration module for testing C-Star with pytest.
+
+The first section sets or gets any globally accessible paths (e.g. that of the test directory itself, TEST_DIRECTORY).
```

Comment:
The test directory just means the directory from which you call `pytest`?

Reply:
It refers to `<top_level>/cstar/tests` , so perhaps should be `tests` directory. I am trying to use absolute paths throughout rather than defining things relative to the directory in which pytest is run.